app/services/album.js

```diff
@@ -22,4 +22,12 @@ exports.checkAlbum = (id) => {
       });
   });
 
+};
+
+exports.getPhotoList = (id) => {
+
+  return new Promise((resolve, reject) => {
+    rp(config.common.urlRequests.photoList + id).then(response => resolve(response));
```

Comment:
Also here, the request promise method is returning a promise, you can return that instead of creating a new one.